components/eventing-controller/README.md

```diff
@@ -29,49 +30,35 @@ This component contains controllers for various CustomResourceDefinitions relate
     ```sh
     make deploy-eventing-controller-nats-local
```

Comment:
In the [installation](https://github.com/kyma-project/kyma/blob/8cfb79ec0b054852b9aaf8cbbe5ec696f76c75ba/components/eventing-controller/README.md#installation) section, we still mention both `controller`  and `nats-controller`, so should `nats-controller` also be removed here?

Reply:
Yes, will be, also the Makefile. That's part of the WiP I have to discuss with @radufa .